cores-and-flashing.tex

```diff
@@ -279,19 +279,23 @@ \section{Understanding The Core Booting Process}
 
 \includegraphics[width=\linewidth]{images/illustrations/flashmenu-flowchart.pdf}
 
-When the MEGA65 is
-powered on, it does the following:
+The booting process is governed by two facilities:
 \begin{itemize}
-\item Starts the bitstream stored in slot 0 of flash memory. If that is the MEGA65 Factory Core, the MEGA65 HYPPO
-      Hypervisor starts.
-\item If it is the first boot since power-on, HYPPO starts the Flash Menu program -- but note that the Flash Menu in
+  \item The Hypervisor (also known as HYPPO), which operates at a level above the Kernal. One of its responsibilities is to manage aspects of the boot process. For more details on the Hypervisor, refer to the {\bf MEGA65 Book}. In the diagram, activities performed by the Hypervisor have been highlighted in green.
+  \item The Flash Menu program (also known as MegaFlash), which provides a list of available core slots to choose from. In the diagram, activities performed by MegaFlash have been highlighted in blue.
+\end{itemize}
+
+When the MEGA65 is powered on, it does the following:
+\begin{itemize}
+\item Loads the bitstream stored in slot 0 of flash memory. If that is the MEGA65 Factory Core, the MEGA65 
+  HYPPO Hypervisor starts.
+\item If it is the first boot since power-on (which implies that we are running from slot 0), HYPPO starts the Flash Menu program (aka MegaFlash) -- but note that the Flash Menu in
```

Comment:
`implies that it is running from slot 0`, or `implies that the MEGA65`...

Reply:
I'm ok with 'we' in this context, as if we're looking through the eyes of the process.